Dockerfile

```diff
@@ -8,13 +8,33 @@ RUN curl -sL https://deb.nodesource.com/setup_4.x | bash -
 RUN apt-get update && apt-get install -y nodejs
 RUN npm install -g npm
 
-# copy in the pip-service code
-WORKDIR /pip
-COPY . .
+# download WOF data
+RUN mkdir /whosonfirst
+
+# copy pelias config
+ADD './docker/pelias.json' '/code/pelias.json'
+ENV PELIAS_CONFIG '/code/pelias.json'
+
+# clone repo
+RUN git clone https://github.com/pelias/whosonfirst.git /code/pelias/whosonfirst
+
+# change working dir
+WORKDIR /code/pelias/whosonfirst
+
+# install npm dependencies
 RUN npm install
+RUN npm run download
```

Comment:
are you sure you want to run this at `build time`? that will mean all the WOF data files will be stored inside the image.

I think it's a better idea if docker **images** only contain code and that the user executes scripts inside a **container** at `run time` to download the data on to an ephemeral/persistent disk of choice.

having a clearer separation of code from data will allow for much more flexibility and will also reduce the size of images and the frequency they need to be rebuilt.

also worth considering that the filtered download options in `pelias.json` will only download a subset of the global data, making any changes or updates to this data would require a new **image** be built, even if the code remained the same.